src/cmdstan/arguments/arg_adapt.hpp

```diff
@@ -18,14 +15,22 @@ class arg_adapt : public categorical_argument {
     _name = "adapt";
     _description = "Warmup Adaptation";
 
-    _subarguments.push_back(new arg_adapt_engaged());
-    _subarguments.push_back(new arg_adapt_gamma());
-    _subarguments.push_back(new arg_adapt_delta());
-    _subarguments.push_back(new arg_adapt_kappa());
-    _subarguments.push_back(new arg_adapt_t0());
-    _subarguments.push_back(new arg_adapt_init_buffer());
-    _subarguments.push_back(new arg_adapt_term_buffer());
-    _subarguments.push_back(new arg_adapt_window());
+    _subarguments.push_back(
+        new arg_single_bool("engaged", "Adaptation engaged?", true));
+    _subarguments.push_back(new arg_single_real_pos(
+        "gamma", "Adaptation regularization scale", 0.05));
+    _subarguments.push_back(new arg_single_real_bounded(
+        "delta", "Adaptation target acceptance statistic", 0.8, 0.1, 0.999));
```

Comment:
These bounds are different than the originals. You can make them equivalent (in the sense of up-to-machine-precision) by using [`std::nextafter`](https://en.cppreference.com/w/cpp/numeric/math/nextafter).

E.g., `std::nextafter(0., 1 /* or any positive number */)` will give you the smallest number greater than 0, such that
`x > 0` is equivalent to `x >= std::nextafter(0.0, 1)`

Similarly, `nextafter(1.0, 0.0)` gives you double value immediately _smaller_ than 1.0

Reply:
will do.
although in practice, at the point where accept delta needs to be 0.999, the model is probably misspecified.

Reply:
followed suggestion.